client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/index.ts

```diff
@@ -42,6 +43,8 @@ export default function useGetThankYouUrl( {
 	isInEditor?: boolean;
 } ): GetThankYouUrl {
 	const selectedSiteData = useSelector( ( state ) => getSelectedSite( state ) );
+	const shouldShowOneClickTreatment = useSelector( ( state ) => isTreatmentOneClickTest( state ) );
```

Comment:
@jessie-ross @withinboredom can this be simplified in light of #46805?

Reply:
I don't think that API is ready for use yet, see #48056 for what might be a long term fix to having a simpler Calypso client.